frontend/src/types/index.ts

```diff
@@ -103,3 +98,24 @@ export interface DirectoryData {
     tooling: Tools;
   };
 }
+
+export enum DatabaseType {
```

Comment:
Why does the front-end need this? 🤔

Reply:
Deleted this enum as it was no longer needed. But there was no other place for types, have now created one on the server side to capture server side only types.

Reply:
awesome :-)